src/TwigComponent/tests/Fixtures/EventListener/FooBarComponentTemplateListener.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent\Tests\Fixtures\EventListener;
+
+use Symfony\Component\EventDispatcher\Attribute\AsEventListener;
+use Symfony\UX\TwigComponent\Event\PreRenderEvent;
+
+/**
+ * Change the template of "FooBarBaz:" components during PrePrenderEvent.
+ * Before:
+ *      FooBar:Baz -> FooBar/Baz.html.twig
+ * After:
+ *      FooBar:Baz -> FooBar/Baz.foo_bar.html.twig
+ *
+ * @see PreRenderEvent
+ */
+class FooBarComponentTemplateListener
```

Comment:
Is this registered in the test kernel and being used?

Reply:
It is registered (the test kernel has autowire+autoconfigure for the entire namespace) and it is used 

There are 4 data provided / tested in src/TwigComponent/tests/Integration/ComponentEventTest.php and only two fail because the FooBarComponentTemplateListener is called